changes/331.general.rst

```diff
@@ -0,0 +1,2 @@
+- Performance improvements for saturation step. (`#331
+  <https://github.com/spacetelescope/stcal/issues/331>`_)
```

Comment:
```suggestion
Performance improvements for saturation step
```
towncrier will add the link and bullet when this changelog fragment is consumed (the next build). Accepting this suggestion should fix the failing docs.